src/diagnostics/logPointsSessionWizard.ts

```diff
@@ -78,6 +79,40 @@ export class LogPointsSessionAttach extends WizardBase {
     }
 }
 
+// tslint:disable:max-classes-per-file
+class EligibilityCheck extends WizardStep {
+    constructor(private _wizard: LogPointsSessionAttach) {
+        super(_wizard, 'Decide the app service eligibility for logpoints.');
+    }
+
+    public async prompt(): Promise<void> {
+        const site = this._wizard.site;
+
+        const kind = site.kind;
+
+        if (!/linux$/.test(kind)) {
+            throw new Error('Only Linux App Services are suppored');
+        }
+
+        const siteClient = this._wizard.websiteManagementClient;
+
+        const config: SiteConfigResource = await siteClient.webApps.getConfiguration(site.resourceGroup, site.name);
+
+        const linuxFxVersion = config.linuxFxVersion;
+
+        if (!linuxFxVersion) {
+            throw new Error('Cannot read "linuxFxVersion"');
+        }
+
+        const [framework, imageName] = linuxFxVersion.split('|');
+        const enabledImages = vscode.workspace.getConfiguration('appService').get<string[]>('enabledDockerImages');
+
+        if ('docker' !== framework.toLocaleLowerCase() || enabledImages.indexOf(imageName.toLocaleLowerCase()) === -1) {
+            throw new Error(`Please use one of the supported docker image. The ${framework}|${imageName} combination is not supported`);

```

Comment:
for the error message, we probably want something that says what are the supported image names.  Or a link to something that lists them out.

Reply:
I'd prefer adding a link. The image names are long. It might not necessarily provide a good experience to user if we enumerate them one by one and cluster the output in a small information prompt. So it's better to use a link. Will add a backlog item.

Reply:
Please put the issue in the wiki - I don't want to lose sight of this before we ship out the MVP.  The concern here is if someone tries to use logpoints on an unsupported image, they have no idea how to determine which instance is supported, and they'll just give up.

Reply:
It is in the wiki.